### DISCUSSION TITLE
Cypress 8.5.0 browser resolves http calls extremely slow

### DISCUSSION THREAD

**Discussion by alexandersels** [cypress-io/cypress#18279] [Questions and Help]
When trying to visit a page in the browser that cypress is spinning up the average response time of my http calls are +- 20s. When I open my native Chrome browser, the website loads under a second.

![image](https://user-images.githubusercontent.com/1790528/135113364-7313c4d7-902d-4085-8208-55a817ae1aff.png)

In the provided screenshot you can see calls that go over 27s of response time ... It seems to be an issue in the browser opened by Cypress (and any browser for that matter).

Anyone able to give me some advice?

Kind regards,

**Comment by beccaonx** [comment]
I have been experiencing a similar issue and It seems that the cypress runner is using HTTP 1.1, which is limited to 6 TCP connections per origin. However, outside cypress, apps may use HTTP 2.0, which does not limit connections.
